charm-slurmctld/config.yaml

```diff
@@ -1,17 +1,57 @@
 options:
-  nagios_context:
-    default: "juju"
+  cluster-name:
     type: string
+    default: osd-cluster
     description: |
-      Used by the nrpe subordinate charms.
-      A string that will be prepended to instance name to set the host name
-      in nagios. So for instance the hostname would be something like:
-          juju-myservice-0
-      If you're running multiple environments with the same services in them
-      this allows you to differentiate between them.
-  nagios_servicegroups:
+      Name to be recorded in database for jobs from this cluster.  This is
```

Comment:
Is there a reason to use that line width? I think the readability is increased if we increase that, maybe 100?

Reply:
That's the default 80 columns. I like this line length for text.

Reply:
Do you have something against changing to 100? I think most of the projects use that, and I don't see the need to limit ourselves to only 80 (and it can cause a lot of unnecessary line breaks)

Reply:
And of course that must not be done in this PR, since it will transform it in a mess, but I'm thinking in a next one to refactor that

Reply:
for text, i prefer ~80 columns. I have difficulty reading and line lengths of ~80 are easier.

I increased the limit on the linter for code though.